src/screens/GroupInfo/elements/ModalAction.js

```diff
@@ -160,7 +160,7 @@ const styles = StyleSheet.create({
   textButton: {
     fontSize: 16,
     fontFamily: fonts.inter[600],
-    color: COLORS.white2
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the textButton has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect the readability and contrast of the text on its background. If `COLORS.white2` was a lighter shade, switching to `COLORS.white` might make the text too bright against light backgrounds.
```diff
-    color: COLORS.white2
+    color: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->